src/interfaces/payment-stripe/methods/coupon/create.ts

```diff
@@ -0,0 +1,20 @@
+import type ICoupon from '../../entities/coupon';
+import type { TCouponDuration, TCurrency } from '../../interfaces';
+
+interface ICreateCouponInput {
+  duration: TCouponDuration;
+  products: string[];
```

Comment:
```suggestion
  productIds: string[];
```

Reply:
endpoint use 'products' property

Reply:
if u provided ids it should be proudct ids, write todo and fix later endpoint input